app/models/celebrity.rb

```diff
@@ -1,2 +1,15 @@
 class Celebrity < ApplicationRecord
+    validates :name, presence: true
+      validates :name, length: { minimum: 2 }
+
+
+         validates :age, presence: true
+
+       validates :stared_in, presence: true
+       validates :stared_in, length: { minimum:10 }
+
+
+    validates :image, presence: true
```

Comment:
Please clean up the indentation and spacing.

Reply:
Fixed it